library/src/main/kotlin/com/oddin/oddsfeedsdk/cache/entity/TournamentCache.kt

```diff
@@ -176,7 +178,8 @@ data class LocalizedTournament(
     var endDate: Date?,
     var sportId: URN,
     var scheduledTime: Date?,
-    var scheduledEndTime: Date?
+    var scheduledEndTime: Date?,
+    var otsRiskTier: Int?
```

Comment:
I do not recommend to use var in data classes.  But I understand that you just followed current style. If you would use val instead of var, then you would have to refactor refreshOrInsertItem method in TournamentCacheImpl to:

`         
item = item.copy(
    startDate = Utils.parseDate(tournament.tournamentLength?.startDate),
    endDate = Utils.parseDate(tournament.tournamentLength?.endDate),
    sportId = URN.parse(tournament.sport.id),
    scheduledTime = Utils.parseDate(tournament.scheduled),
    scheduledEndTime = Utils.parseDate(tournament.scheduledEnd),
)
`
I do not say refactor it just let say some recommendation for new classes :)